lib/puppet_x/chocolatey/chocolatey_install.rb

```diff
@@ -13,7 +17,7 @@ def self.install_path
             hive.open('SYSTEM\CurrentControlSet\Control\Session Manager\Environment', Win32::Registry::KEY_READ | 0x100) do |reg|
               value = reg['ChocolateyInstall']
             end
-          rescue Win32::Registry::Error => e
+          rescue Win32::Registry::Error
```

Comment:
This rescue should probably be logged when the registry key cannot be found or read.